schemas/openapi.json

```diff
@@ -2024,25 +2036,25 @@
         }
       }
     },
-    "/v1/projects/{project_id}": {
+    "/v1/projects/{project_identifier}": {
       "get": {
         "tags": [
           "projects"
         ],
-        "summary": "Get project by ID",
-        "description": "Retrieve a specific project using its unique identifier.",
+        "summary": "Get project by ID or name",
+        "description": "Retrieve a specific project using its unique identifier: either project ID or hex-encoded project name.",
         "operationId": "getProject",
         "parameters": [
           {
-            "name": "project_id",
+            "name": "project_identifier",
             "in": "path",
             "required": true,
             "schema": {
               "type": "string",
-              "description": "The ID of the project.",
-              "title": "Project Id"
+              "description": "The project identifier: either project ID or hex-encoded project name.",
```

Comment:
hmmm, the ergonomics of the hex encoding seems non-ideal. Why not just best effort for the name - if it's non URL safe we just say it's not possible?